Tests/Marketplace/zip_packs.py

```diff
@@ -56,6 +57,8 @@ def zip_packs(packs, destination_path):
     with ZipFile(os.path.join(destination_path, ARTIFACT_NAME), mode='w') as zf:
         for zip_pack in packs:
             for name, path in zip_pack.items():
+                if name not in packs_to_include:
```

Comment:
`packs_to_include` might be empty as it is not a required argument.
add a check like [here](https://github.com/demisto/content/pull/8741/files?file-filters%5B%5D=#diff-07f5616b9b8d97b7b823933458483095R131)